lib/cfg-lex.l

```diff
@@ -338,6 +337,13 @@ word	[^ \#'"\(\)\{\}\[\]\\;\r\n\t,|\.@:]
 			     g_string_truncate(yyextra->string_buffer, 0);
 			     yy_push_state(qstring, yyscanner);
 			   }
+\.			   {
```

Comment:
Can we extend `({word}+(\.)?)*{word}+` as well to accept leading/trailing dots?
For example: `((\.)?{word}+)*{word}+(\.)?`

Reply:
That is a slippery slope, there is L_DOTDOT, and L_DOTDOTDOT rule in the lexer.

Reply:
`L_DOTDOT` and `L_DOTDOTDOT` has priority over this rule, but my example does not allow more than one dot between "word"s afaik.

Reply:
Sure it works, not intended to say otherwise. Only that makes `.SDATA.` okay, but `..SDATA` not, while the later in the realm of syslog-ng not that desired. I would not extend now the stringification with. I have no strong argument for either side.